src/ui/index.html

```diff
@@ -1,15 +1,24 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>ZORI - Loading&hellip;</title>
-        <style>
-            @tailwind base;
-            @tailwind components;
-            @tailwind utilities;
-        </style>
-    </head>
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>ZORI - Loading&hellip;</title>
+
+    <!--Adobe fonts-->
+    <link rel="stylesheet" href="https://use.typekit.net/azl7eqx.css">
+
+    <!--Google fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300..900;1,300..900&display=swap" rel="stylesheet">
```

Comment:
This is a lot of style info to load for this one font. You sure you need all of those weights and configurations?

Reply:
Probably not? I'll check Google fonts again and see what is needed.